src/main/java/org/example/AddBadge.java

```diff
@@ -3,24 +3,21 @@
 import net.dv8tion.jda.api.events.message.MessageReceivedEvent;
 
```

Comment:
Did you use the right file is seems there is a difference in the code. Is this a older version?
![image](https://user-images.githubusercontent.com/78591393/184243685-679301dc-4b6e-4c46-b619-16ccd1471fbc.png)